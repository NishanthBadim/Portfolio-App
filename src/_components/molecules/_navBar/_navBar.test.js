import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import NavBar from "./_navBar";
import { BrowserRouter } from "react-router-dom";

describe("Navigation Bar testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<BrowserRouter><NavBar /></BrowserRouter>);
    expect(input.toJSON()).toMatchSnapshot();
  })
})