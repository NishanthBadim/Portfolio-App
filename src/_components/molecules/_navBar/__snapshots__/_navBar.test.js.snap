// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Bar testing Matches the snapshot 1`] = `
<header
  className="mdl-layout__header mdl-layout__header--scroll mdl-layout__header--transparent makeStyles-head-2"
>
  <div
    className="mdl-layout__header-row"
  >
    <span
      className="mdl-layout-title"
    >
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": "2vw",
            "textDecoration": "none",
          }
        }
      >
        My Portfolio
      </a>
    </span>
    <div
      className="mdl-layout-spacer"
    />
    <nav
      className="mdl-navigation"
    >
      <a
        className="mdl-navigation__link"
        href="/"
        onClick={[Function]}
      >
        <p
          className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
        >
          Home
        </p>
      </a>
      <a
        className="mdl-navigation__link"
        href="/gallery"
        onClick={[Function]}
      >
        <p
          className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
        >
          Gallery
        </p>
      </a>
      <a
        className="mdl-navigation__link"
        href="/admin"
        onClick={[Function]}
      >
        <p
          className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
        >
          Admin
        </p>
      </a>
      <a
        className="mdl-navigation__link"
        href="/aboutme"
        onClick={[Function]}
      >
        <p
          className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
        >
          About Me
        </p>
      </a>
      <a
        className="mdl-navigation__link"
        href="/contact"
        onClick={[Function]}
      >
        <p
          className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
        >
          Contact
        </p>
      </a>
    </nav>
  </div>
</header>
`;
